### PR TITLE
Example with maven forks provided

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: example-testng-fork-execution/src/main/resources/reportportal.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+rp.uuid = 
+
+rp.launch = superadmin_TEST_EXAMPLE
+
+rp.project = superadmin_personal
+rp.reporting.async=true
+rp.client.join=true
+rp.client.join.lock.file.name=/Users/yumfriez/qwe/rp.lock
+rp.client.join.sync.file.name=/Users/yumfriez/qwe/rp.sync
```
Comment (ihar-kahadouski) [inline example-testng-fork-execution/src/main/resources/reportportal.properties]: /Users/yumfriez/qwe/ ???
Can we use some file inside resources and relative path to it?

**Review by ihar-kahadouski — CHANGES_REQUESTED** [review]

File: example-testng-fork-execution/src/main/resources/reportportal.properties (hunk 1 above)
Comment (HardNorth) [reply example-testng-fork-execution/src/main/resources/reportportal.properties]: +1 this is absolutely unnecessarily to put full path. I beleive these files will be created in build directory if just file name specified.

File: example-testng-fork-execution/suites/logging_tests.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Logging Tests" parallel="classes" thread-count="10">
```
Comment (HardNorth) [inline example-testng-fork-execution/suites/logging_tests.xml]: Do we really need to additionally check that in a multi-thread execution?

File: example-testng-fork-execution/src/main/resources/reportportal.properties [hunk 3]
```diff
@@ -0,0 +1,11 @@
+rp.endpoint =
+
+rp.uuid = 
+
+rp.launch = superadmin_TEST_EXAMPLE
+
+rp.project = superadmin_personal
```
Comment (HardNorth) [inline example-testng-fork-execution/src/main/resources/reportportal.properties:4]: Please apply common format for the fields above.

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: example-testng-fork-execution/src/main/resources/reportportal.properties (hunk 1 above)
Comment (Yumfriez) [reply example-testng-fork-execution/src/main/resources/reportportal.properties]: fixed

File: example-testng-fork-execution/suites/logging_tests.xml (hunk 2 above)
Comment (Yumfriez) [reply example-testng-fork-execution/suites/logging_tests.xml]: removed

File: example-testng-fork-execution/src/main/resources/reportportal.properties (hunk 3 above)
Comment (Yumfriez) [reply example-testng-fork-execution/src/main/resources/reportportal.properties:4]: fixed

**Review by HardNorth — APPROVED** [review]

**Pull request merged**